jax_unirep/utils.py

```diff
@@ -160,49 +188,50 @@ def get_embeddings(sequences: List[str]) -> np.ndarray:
     return onp.stack(seq_embeddings, axis=0)
 
 
-def load_dense_1900(name: str = "uniref50") -> Dict:
+def load_dense_1900(folderpath: Optional[str] = None) -> Tuple:
     """
     Load pre-trained dense layer weights from the UniRep paper.
 
     The dense layer weights are used to predict next character
     from the output of the mLSTM1900.
     """
+    if folderpath:
+        weights_1900_dir = Path(folderpath)
+    else:
+        weights_1900_dir = Path(
+            pkg_resources.resource_filename(
+                "jax_unirep", "weights/1900_weights/uniref50"
+            )
+        )
```

Comment:
This looks duplicated. Can we have a single source of truth the default weights dir, wrapped in a function?